lib/helpers/mail/model/MailSettings.php

```diff
@@ -0,0 +1,76 @@
+<?php namespace SendGrid\Helpers\Mail\Model;
+
+class MailSettings implements \JsonSerializable
+{
+    private $bcc;
+    private $bypass_list_management;
+    private $footer;
+    private $sandbox_mode;
+    private $spam_check;
+
+    public function setBccSettings($bcc)
+    {
+        $this->bcc = $bcc;
+    }
+
+    public function getBccSettings()
+    {
+        return $this->bcc;
+    }
+
+    public function setBypassListManagement($bypass_list_management)
+    {
+        $this->bypass_list_management = $bypass_list_management;
+    }
+
+    public function getBypassListManagement()
+    {
+        return $this->bypass_list_management;
+    }
+
+    public function setFooter($footer)
+    {
+        $this->footer = $footer;
+    }
+
+    public function getFooter()
+    {
+        return $this->footer;
+    }
+
+    public function setSandboxMode($sandbox_mode)
```

Comment:
Random suggestion, what do you think about `enableSandboxMode()` and `disableSandboxMode()`, which will do `$this->sandbox_mode = true` and `$this->sandbox_mode = false` respectively,  instead of boolean parameter?

Reply:
👍